misc/incremental_checker.py

```diff
@@ -0,0 +1,218 @@
+#!/usr/bin/env python
```

Comment:
Should probably be python3. Also can you make the file executable (chmod +x) so the #! actually works?